crates/tx5-signal/Cargo.toml

```diff
@@ -31,8 +31,6 @@ tokio-tungstenite = { workspace = true }
 tracing = { workspace = true }
 tx5-core = { workspace = true }
 url = { workspace = true }
-
-[target.'cfg(not(any(target_os = "windows", target_os = "linux", target_os = "macos")))'.dependencies]
```

Comment:
This doesn't appear to be platform specific in any way, but it looks like it was deliberately only included for platforms that won't have a certificate store. Now that we're successfully using the certificate store on windows and this isn't being used on the platforms it's been included for, should this be reverted?

Reply:
Yes, this should be reverted. `rustls_native_certs` doesn't currently work on android or ios. (although it'd be nice if it did : )

Reply:
Oh yes, sorry. I added this with the initial approach where the certs would be bundled.